### PR TITLE
drag drop, swipe in now playing lists + refresh in genres activity

### PR DESCRIPTION
**Pull request by vu9596**
drag drop to move track, swipe to delete track in now playing lists + refresh in genres activity

### CODE REVIEW

**Comment by vu9596** [comment]
ready

File: app/src/main/res/layout/layout_play_info.xml [hunk 1]
```diff
@@ -71,7 +71,7 @@
     <TextView
         android:id="@+id/text_artist"
         android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
+        android:layout_height="20dp"
```
Comment (NguyenMinhTuanB) [inline app/src/main/res/layout/layout_play_info.xml]: hardcode

File: app/src/main/res/layout/activity_genres.xml [hunk 2]
```diff
@@ -79,6 +85,6 @@
             app:layout_constraintBottom_toTopOf="@id/mini_player"
             app:layout_constraintEnd_toEndOf="parent"
             app:layout_constraintStart_toStartOf="parent"
-            app:layout_constraintTop_toBottomOf="@id/recycler_list_tracks" />
+            app:layout_constraintTop_toBottomOf="@id/swiperefresh" />
```
Comment (NguyenMinhTuanB) [inline app/src/main/res/layout/activity_genres.xml:88]: line 73. not match_parent

**Comment by NguyenMinhTuanB** [comment]
commented

**Comment by vu9596** [comment]
ready

**Review by NguyenMinhTuanB — APPROVED** [review]

**Pull request merged**